composer.json

```diff
@@ -23,8 +23,9 @@
     "php" : "^7.0",
     "illuminate/contracts": "~5.1.0|~5.2.0|~5.3.0",
     "illuminate/support": "~5.1.0|~5.2.0|~5.3.0",
-    "google/apiclient" : "^1.1",
-    "nesbot/carbon": "^1.21"
+    "google/apiclient" : "^2.0",
+    "nesbot/carbon": "^1.21",
+    "tedivm/stash": "^0.14.1"
```

Comment:
There's a bridge already, [illuminate-psr-cache-bridge](https://github.com/madewithlove/illuminate-psr-cache-bridge), but since you don't want a new dependency should I just copy the implementation to this package ?